tests/hash_test.cpp

```diff
@@ -75,9 +75,11 @@ void put_coordinate( std::unordered_set<tripoint, Hash> &c, int x, int y )
 }
 
 // These are the old versions of the hash functions for point and tripoint.
```

Comment:
Maybe just nuke these instead, there's 0 chance we're restoring these old hash functions.

Reply:
I was thinking about that.  Done.